### PR TITLE
add mineleague

### PR DESCRIPTION
**Pull request by soeinyoshi**
## Type of change

- [x] Add new directory for a server
- [ ] Update directory of a server
- [ ] Documentation Update

## Checklist

- [x] I have read the [README](https://github.com/LabyMod/server-media/blob/master/README.md) doc
- [X] I have compressed the images appropriately (e.g. on https://tinypng.com)
- [X] I have created the manifest.json with the required values

### CODE REVIEW

**Comment by soeinyoshi** [comment]
add mineleague

File: minecraft_servers/mineleague/manifest.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+   "nice_name":"MineLeague.eu",
+   "direct_ip":"mineleague.eu",
+   "supported_languages":[
+	  "de",
+	  "en",
+	  "al"
+   ],
+   "social":{
+      "web":"https://mineleague.eu",
+      "discord":"https://link.mineleague.eu/discord"
```
Comment (JakobBruening) [inline minecraft_servers/mineleague/manifest.json:12]: As custom discord invite links are only supported for LabyMod partners at the moment.
Please use a discord invite link.

Documentation reference: https://github.com/LabyMod/server-media/blob/master/docs/Manifest.md#discord-url

**Review by JakobBruening — CHANGES_REQUESTED** [review]
Please compress all files, especially the background@2x.png is too large.

**Comment by soeinyoshi** [comment]
fixed

**Review by JakobBruening — CHANGES_REQUESTED** [review]
Invalid workflow.json - Fix json!

**Comment by JakobBruening** [comment]
I will close the PR due to inactivity.
Please re-open or re-create as soon as the changes are applied to your fork.

**Pull request closed without merging**